src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/OffenderService.kt

```diff
@@ -36,6 +43,33 @@ OffenderService(
     offenderLicences = offenderLicences.map { it.copy(responsibleCom = newCom) }
     this.licenceRepository.saveAllAndFlush(offenderLicences)
 
+    if (offenderLicences.any {
+      it.statusCode === LicenceStatus.NOT_STARTED || it.statusCode === LicenceStatus.IN_PROGRESS
+    }
+    ) {
+      val releaseDate = offenderLicences[0].actualReleaseDate ?: offenderLicences[0].conditionalReleaseDate
+      if (LocalDate.now() >= this.releaseDateService.getEarliestReleaseDate(
```

Comment:
I am trying to get 5th working day before release date and then compare it with current date.
The number I am passing as **maxNumberOfWorkingDaysAllowedToTriggerEmailIfPPIsModifiedOrAllocatedBefore** is 6 as we are excluding the current day.  

If the returned date is less than equal to current date then it means the PP is allocated/reassigned before 5 working days and we are triggering an email.